gbpservice/neutron/nsf/proxy/cfg.py

```diff
@@ -0,0 +1,5 @@
+OPTS = [
+    {'thread_pool_size': 10},
+    {'unix_bind_path': '/tmp/uds_socket'},
```

Comment:
This file is not part of config_agent ,removed core and proxy from config_agent_modules branch